components/signup/signupform.tsx

```diff
@@ -177,18 +178,26 @@ export default function SignUpForm() {
                                 onChange={(e) => setEmail(e.target.value)}
                             />
                         </div>
-                        <div className="mb-4">
-                            <Label htmlFor="password" >Password</Label>
+                        <div className="mb-4 relative">
+                            <Label htmlFor="email">Password</Label>
+
                             <Input
-                                type="password"
+                                type={showPassword ? "text" : "password"}
                                 id="password"
-                                placeholder="Password"
+                                placeholder="password"
                                 value={password}
-                                onChange={(e) => {
-                                    setPasswordStrength(e.target.value.length);
-                                    setPassword(e.target.value);
-                                }}
+                                onChange={(e) => setPassword(e.target.value)}
+                                className="pl-3 pr-10 py-2 rounded-lg"
```

Comment:
All this class name are not necessary we already have those defined in global component/ui.

Kindly remove unwanted class names